### ISSUE TITLE
Running in multiple gpus

### ISSUE THREAD

**Issue by rdiazgar** [bonlime/keras-deeplab-v3-plus#64]
Hi,

Has anyone trained this model using multiple gpus? I am running into problems when doing so. I'm using the standard multi_gpu_model wrapper for this. The first training epoch seems to converge normally, but the validation loss and the following epochs obtain much higher loss values and never recover from this.

Best
Raul

**Comment by trungpham2606** [comment]
@rdiazgar  Can you provide some results images after fine-tuning the model in this repo :-? 
And your keras version also. 
Thanks in advance

**Comment by rdiazgar** [comment]
@trungpham2606 , as I mentioned in #56 , I can't share results as of now.

My Keras version is 2.2.4, using TF 1.12.0 as backend.

Raul

**Comment by rdiazgar** [comment]
It looks like the problem is solved by setting the multi_gpu_model wrapped with cpu_merge=False.

Does anyone have any intuition why this works. From the keras documentation, it seems that users are encouraged to set cpu_merge=True. I was wondering if anyone else had tried this repu with multiple gpus and what their opinion was.

Thanks
Raul
